infrastructure/variables.tf

```diff
@@ -15,3 +15,15 @@ variable "deployment_namespace" {}
 variable "common_tags" {
   type = map(string)
 }
+
+variable "postgres_db_component_name" {
+  default = "cft-case-event-handler"
```

Comment:
This should probably be: `wa-case-event-handler`

Reply:
done